install.sh

```diff
@@ -30,56 +34,50 @@ else
   USER_SPACE_ARCH="${ARCH}"
 fi
 
-# BOOTSTRAP_PACKAGES cannot depend on crew_profile_base for their core operations (completion scripts are fine)
-BOOTSTRAP_PACKAGES="crew_mvdir pixz ca_certificates ruby openssl"
-[ -x /usr/bin/zstd ] || BOOTSTRAP_PACKAGES="zstd ${BOOTSTRAP_PACKAGES}" # use system zstd if available
-[ -x "${CREW_PREFIX}"/bin/xz ] && rm "${CREW_PREFIX}"/bin/xz # remove local xz if installed
-
-RED='\e[1;91m';    # Use Light Red for errors.
-YELLOW='\e[1;33m'; # Use Yellow for informational messages.
-GREEN='\e[1;32m';  # Use Green for success messages.
-BLUE='\e[1;34m';   # Use Blue for intrafunction messages.
-GRAY='\e[0;37m';   # Use Gray for program output.
-MAGENTA='\e[1;35m';
+if [[ "$ARCH" == "x86_64" ]]; then
+  LIB_SUFFIX='64'
```

Comment:
Isn't `LIB_SUFFIX` also 64 on aarch64 systems with a 64-bit userland?

Reply:
But we're distributing 32-bit libraries on them, no?

Reply:
Yes, but from what I've been noticing some of the newer aarch64 systems are implementing a 64-bit userland which will necessitate aarch64 binaries eventually.

Reply:
Given that when we do support aarch64 binaries it'll require a complete reinstallation, we can update install.sh when we add support.